sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/package-info.java

```diff
@@ -1,14 +1,530 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-// Code generated by Microsoft (R) AutoRest Code Generator.
-
 /**
- * Package containing classes for creating clients {@link
- * com.azure.security.keyvault.administration.KeyVaultAccessControlAsyncClient} and {@link
- * com.azure.security.keyvault.administration.KeyVaultAccessControlClient} that perform access control operations on
- * Azure Key Vault resources, as well as clients {@link
- * com.azure.security.keyvault.administration.KeyVaultBackupAsyncClient} and {@link
- * com.azure.security.keyvault.administration.KeyVaultBackupClient} that perform backup and restore operations on Azure
- * Key Vault keys.
+ * <a href="https://learn.microsoft.com/azure/key-vault/managed-hsm/">Azure Key Vault Managed HSM</a> is a
+ * fully-managed, highly-available, single-tenant, standards-compliant cloud service that enables you to safeguard
+ * cryptographic keys for your cloud applications using FIPS 140-2 Level 3 validated HSMs.
+ *
+ * <p>
+ * The Azure Key Vault Administration client library allows developers to interact with the Azure Key Vault Managed
+ * HSM service from their applications. The library provides a set of APIs that enable developers to perform
+ * administrative tasks such as full backup/restore, key-level role-based access control (RBAC), and account settings
+ * management.
+ *
+ * <p>
+ * <strong>Key Concepts:</strong>
+ *
+ * <p>
+ * <strong>What is a Key Vault Access Control Client?</strong>
+ * <p>
+ * The Key Vault Access Control client performs the interactions with the Azure Key Vault service for getting,
+ * setting, deleting, and listing role assignments, as well as listing role definitions. Asynchronous
+ * (KeyVaultAccessControlAsyncClient) and synchronous (KeyVaultAccessControlClient) clients exist in the SDK allowing
+ * for the selection of a client based on an application's use case. Once you've initialized a role assignment, you can
+ * interact with the primary resource types in Key Vault.
+ *
+ * <p>
+ * <strong>What is a Role Definition?</strong>
+ * <p>
+ * A role definition is a collection of permissions. It defines the operations that can be performed, such as read,
+ * write, and delete. It can also define the operations that are excluded from allowed operations.
+ *
+ * <p>
+ * Role definitions can be listed and specified as part of a role assignment.
+ *
+ * <p>
+ * <strong>What is a Role Assignment?</strong>
+ * <p>
+ * A role assignment is the association of a role definition to a service principal. They can be created, listed,
+ * fetched individually, and deleted.
+ *
+ * <p>
+ * <strong>What is a Key Vault Backup Client</strong>
+ * <p>
+ * The Key Vault Backup Client provides both synchronous and asynchronous operations for performing full key backups,
```

Comment:
This section is worded slightly different. Does the Key Vault Backup Client have both sync and async operations in both the sync and async clients or is this referring to the existence of the sync and async clients?